recipes

```diff
@@ -162,6 +162,30 @@ function core_tests()
     python3 -m pip install -U "${CORE_TESTS_PIP_DEPENCENDIES[@]}"
 }
 
+function demo()
+{
+    DOMAIN="demo.yunohost.org"
```

Comment:
So that hard-codes the domain… would there be a solution to make it configurable ?
(Maybe a change-url afterwards…)

Reply:
Yeah I don't know either, that's not a simple problem, let's go with this for now ...